src/main/webapp/js/adminHome.es6

```diff
@@ -16,18 +16,39 @@
  * @param {String} status
  * @returns {String} a HTML row of action result table
  */
+ /**
+* Substring used to check if join link was created
+*/
+const successStatus = 'successfully created with join link:';
+/* eslint-disable no-param-reassign */
 function createRowForResultTable(shortName, name, email, institution, isSuccess, status) {
+    const linkIndex = status.indexOf(successStatus);
```

Comment:
Making booleans explicit would make the intent much clearer here. Consider using:
```js
const linkIndex = status.indexOf(successStatus);
const isSuccessStatus = linkIndex !== 1;
if (isSuccessStatus) {
    ...
}
```